docs/pages/versions/unversioned/sdk/symbols.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Symbols
+description: A library that allows access to native symbols.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-symbols'
+packageName: 'expo-symbols'
+iconUrl: '/static/images/packages/expo-symbols.png'
+platforms: ['ios']
+---
+
+import APISection from '~/components/plugins/APISection';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+
+> **warning** This is library is currently in beta and subject to breaking changes.
+
+**`expo-symbols`** provides access to the [SFSymbols](https://developer.apple.com/sf-symbols/) library on iOS.
+
+<ImageSpotlight
+  style={{ width: 200 }}
+  src="/static/images/symbols.png"
+  alt="Screenshots of the store review API in action on iOS"
+/>
+
+## Installation
+
+<APIInstallSection packageName="expo-symbols" />
+
+## Usage
+
+```ts
```

Comment:
````suggestion
```jsx App.js
````

I don't see any typescript code being used in the code block below. Let's use `jsx` as the code language for this snippet so that we maintain the consistency when mentioning code languages for code blocks in our docs. Also, let's add a file name.